LabExT/Movement/Stage.py

```diff
@@ -63,16 +63,40 @@ class Stage(ABC):
     meta = StageMeta("", False)
 
     @classmethod
-    def find_stage_classes(cls, subdir="Stages") -> list:
+    def find_stage_classes(
+        cls,
+        search_paths: list = [],
```

Comment:
Could this cause the issue that paths are re-inserted even if they are already there?

Reply:
Should work now, I do the import in the new `import_stage_classes` method of the mover. It will be invoked once by the setup runner, similar to measurement etc